samples/package.json

```diff
@@ -12,7 +12,7 @@
         "typescript": "^4.1.3"
     },
     "devDependencies": {
-        "@azure/identity": "^1.5.2",
+        "@azure/identity": "^3.1.2",
```

Comment:
This is a big jump, any risks here?

Reply:
Glanced at the changelog and didn't see any breaking changes that would affect us. This package is used to get the test account credentials for the E2E tests so if something did break the tests on this PR would fail.